### DISCUSSION TITLE
Does Intellichem <--> EasyTouch need to be working prior to install?

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#454] [Q&A] [answered]
I have an issue in the wiring between the EasyTouch and Intellichem.  I need to re-wire between them.

Can I install poolController and dashPanel before I redo the wiring between Intellichem and EasyTouch?  Then, once wiring is fixed, I can communicate with Intellichem?

Or,, will I have to redo install once I've redone the wiring?

**Comment by rstrouse** [comment, marked as answer]
You can add or remove equipment to the configuration at any time.  This includes IntelliChem.  It doesn't matter to njsPC.

**Reply by davthomaspilot** [reply]
Thanks!  Got a new rpi, Bullseye, nodejs-poolController|dashPanel up and running.  Easy Peasy!
